pkg/cmd/kafka/describe/describe.go

```diff
@@ -125,7 +127,11 @@ func runDescribe(opts *Options) error {
 		}
 	}
 
-	return printKafka(kafkaInstance, opts)
+	if !opts.bootstrapServer {
```

Comment:
I'd rather see `if opts.bootstrapServer` and print the url and return, leaving the entire object as the default.